crates/common/fmt/src/dynamic.rs

```diff
@@ -36,7 +36,13 @@ impl DynValueFormatter {
                 f.write_str("]")
             }
             DynSolValue::Tuple(values) => self.tuple(values, f),
-            DynSolValue::String(inner) => write!(f, "{inner:?}"), // escape strings
+            DynSolValue::String(inner) => {
+                if self.raw {
+                    write!(f, "{inner}")
+                } else {
+                    write!(f, "{inner:?}") // escape strings
```

Comment:
i think we want to escape regardless, just without the quotes, so this should be '.escape_debug()' i believe, without the if